glide.yaml

```diff
@@ -41,3 +41,6 @@ import:
   version: 8e7dc108ab3a1ab6ce6d922bbaff5657b88e8e49
   repo: https://github.com/spf13/pflag
   vcs: git
+- package: github.com/deis/controller-sdk-go
+  repo: https://github.com/joshua-anderson/controller-sdk-go
```

Comment:
this should be changed after https://github.com/deis/controller-sdk-go/pull/69 is merged